lib/vfs/Cargo.toml

```
@@ -0,0 +1,22 @@
+[package]
+name = "wasmer-vfs"
+version = "2.0.0"
+description = "Wasmer Virtual FileSystem"
+authors = ["Wasmer Engineering Team <engineering@wasmer.io>"]
+license = "MIT"
+edition = "2018"
+
+[dependencies]
+libc = { version = "^0.2", default-features = false, optional = true }
+thiserror = "1"
+tracing = { version = "0.1" }
+typetag = { version = "0.1", optional = true }
+serde = { version = "1.0", default-features = false, features = ["derive"], optional = true }
+
+[features]
```

```suggestion
[features]
default = ["host_fs"]
```

Let's define `host_fs` as the default file system, which is already the case and the expected behavior.

---

See https://github.com/wasmerio/wasmer/pull/2523 to solve this comment.